notebook/.gitignore

```diff
@@ -1,4 +1,4 @@
 /deployment.yaml
 /notebook-image
 /notebook-worker-image
-
+/notebook-worker-images
```

Comment:
Why is this a part of the PR?

Reply:
I discovered the broken ignore file when I ran git status on this branch.

moved to https://github.com/hail-is/hail/pull/5309